app/src/main/java/edu/uc/kovaciad/slicetracker/ui/main/MainFragment.kt

```diff
@@ -45,4 +40,9 @@ class MainFragment : Fragment() {
 
     }
 
+    companion object {
+        fun newInstance() = MainFragment()
+    }
+
+
```

Comment:
This is the correct format for class layouts.
check out: 
https://kotlinlang.org/docs/coding-conventions.html